docker-compose.yml

```diff
@@ -1,30 +1,30 @@
-version: '3'
+version: "3"
 
 services:
   server:
-    container_name: server 
+    container_name: server
     build: ./server
-    image: mentor-mentor-platform-server
+    image: railsgirlscph/mentormenteeplatform-server:latest
     environment:
       - POSTGRES_HOST=postgres
       - POSTGRES_DB=mentormentee
       - POSTGRES_PASSWORD=${MY_SECRET_KEY}
       - POSTGRES_USER=mentoruser
 
     ports:
-      - '3000:3000'
+      - "3000:3000"
     volumes:
       - ./server/:/app
     tmpfs:
       - /app/tmp
   my-app:
```

Comment:
Will you change my-app, please, to something better reflecting what it is, e.g. web, ui, frontend or something of sorts? It just annoys my inner nazi :D It is ok to have another pr for that, up to you.

Reply:
Yeah I also think it should have a better name, I'll change the name of the react docker container to ui. 
Do you think we should change the folder structure?

Reply:
Yes, then my mind will relax :) hopefully it won't be too much trouble